### PR TITLE
Long lines

### PR DESCRIPTION
**Pull request by nesrv**

### CODE REVIEW

File: InkscapeExtension/ExtendedHatchFill.py [hunk 1]
```diff
@@ -96,8 +101,109 @@ def recursivelyTraverseSvg(self, a_node_list):
...
-        return 'M {0:f},{1:f} l {2:f},{3:f} '.format(pt1[0], pt1[1], pt2[0] - pt1[0], pt2[1] - pt1[1])
+         x1,y1 = pt1[0],pt1[1]
+         x2,y2 = pt2[0],pt2[1]
 
+         od = self.options.outer_dist # Outer contour indent
+         cc = self.options.curv_coef # Curvature coefficient (steepness angle)                   
+         angle = math.radians(self.options.hatchAngle)
+
+          # absolute length of the segment between the points
+         totLen = ((x2 - x1) ** 2 + (y2 - y1) ** 2) ** 0.5
```
Comment (toxa23) [inline InkscapeExtension/ExtendedHatchFill.py]: please replace "x2 -x1" everywhere in the function to "w" defined on line 113. Same is for "y2 - y1".

File: InkscapeExtension/ExtendedHatchFill.py [hunk 2]
```diff
@@ -96,8 +101,109 @@ def recursivelyTraverseSvg(self, a_node_list):
...
+         r1 = randint(-cc, cc)  
+         r2 = randint(-cc, cc)
+         r3 = randint(-cc, cc)
+         r4 = randint(-cc, cc)
+
+
+         if self.options.mode == "linear":
+            path = f'M {x1 + r1 + od * wx}, {y1 + r2 + od * hx} l {x2-x1 -r3 - 2 * od * wx}, {y2-y1 - r4 - 2 * od * hx}'
+         
+         if self.options.mode == "quadratic":
```
Comment (toxa23) [inline InkscapeExtension/ExtendedHatchFill.py]: I think it makes sense to replace 2nd and following "if"s with "elif". Do you agree?

File: InkscapeExtension/ExtendedHatchFill.py [hunk 3]
```diff
@@ -96,8 +101,109 @@ def recursivelyTraverseSvg(self, a_node_list):
...
+         if self.options.mode == "wavy":
+             path = ''
+             number_segm = self.options.number_segm
+             dx = (x2 - x1 - 2 * od * wx) / number_segm
+             dy = (y2 - y1 - 2 * od * hx) / number_segm
+             for _ in range(number_segm):
+                 r1 = randint(1, cc)
+                 r2 = randint(-cc, 0)
+                 path += f'M {x1 + od * wx}, {y1 + od * hx} \
+                             c {dx / 3}, {5} \
```
Comment (toxa23) [inline InkscapeExtension/ExtendedHatchFill.py]: what is 5? Is it a part of the formula or a constant you found empirically? Does it work for any values of wx, dx? Anyway, please write it without {}

File: InkscapeExtension/ExtendedHatchFill.py [hunk 4]
```diff
@@ -96,8 +101,109 @@ def recursivelyTraverseSvg(self, a_node_list):
...
+             for _ in range(int(number_segm)+1):
+                 r1 = randint(1, cc)
+                 r2 = randint(1, cc)
+                 r3 = randint(1, cc) * 2
+                 r4 = randint(1, cc) * 2
+                 r5 = randint(-cc, cc)
+                 r6 = randint(-cc, cc)
+                 r7 = randint(-cc, cc)
+                 r8 = randint(-cc, cc)
+                 path += f'M {x1 }, {y1 } c {r1},{r3}  {-r + r5}, {r/2 + r7} {-r} {0} \
```
Comment (toxa23) [inline InkscapeExtension/ExtendedHatchFill.py]: please change {0} to 0

**Review by toxa23 — CHANGES_REQUESTED** [review]

File: InkscapeExtension/ExtendedHatchFill.py (hunk 2 above)
Comment (nesrv) [reply InkscapeExtension/ExtendedHatchFill.py]: Yes, of course!

File: InkscapeExtension/ExtendedHatchFill.py (hunk 3 above)
Comment (nesrv) [reply InkscapeExtension/ExtendedHatchFill.py]: Hatch 'wavy line' in progress. Here will be a fixed or random wave height

File: InkscapeExtension/ExtendedHatchFill.py (hunk 4 above)
Comment (nesrv) [reply InkscapeExtension/ExtendedHatchFill.py]: here instead of 0 will be outer contour indent

File: InkscapeExtension/ExtendedHatchFill.py [hunk 5]
```diff
@@ -96,7 +102,91 @@ def recursivelyTraverseSvg(self, a_node_list):
...
     def buildHatchLinePath(self, pt1, pt2):
-        return 'M {0:f},{1:f} l {2:f},{3:f} '.format(pt1[0], pt1[1], pt2[0] - pt1[0], pt2[1] - pt1[1])
+        x1, y1 = pt1[0], pt1[1]
+        x2, y2 = pt2[0], pt2[1]
+
+        od = self.options.outer_dist  # Outer contour indent
+        cc = self.options.curv_coef  # Curvature coefficient (steepness angle)
+
+        # absolute length of the segment between the points
+        totLen = ((x2 - x1) ** 2 + (y2 - y1) ** 2) ** 0.5
```
Comment (toxa23) [inline InkscapeExtension/ExtendedHatchFill.py:112]: The requested change (replace all "x2 - x1" with "w" defined on the line 113, same is for "y2-y1" -> "h" from line 114) is not resolved. Please either make the change or explain why it shouldn't be applied.

**Review by toxa23 — CHANGES_REQUESTED** [review]

**Pull request merged**